src/Theme/__stories__/palette.stories.tsx

```diff
@@ -17,34 +18,40 @@ storiesOf('Theme', module)
   })
   .add('Palette', () => {
     return (
-      <Grid>
-        {Object.entries(Theme.palette).map(palette => {
-          const colorName = palette[0];
-          const colorValue = palette[1];
-          if (typeof colorValue == 'function') {
-            return null;
-          }
-          if (typeof colorValue == 'object') {
-            return Object.entries(colorValue).map(subPalette => {
-              const subColorName = subPalette[0];
-              const subColorValue = subPalette[1];
-              return (
-                <PaletteCard
-                  key={subColorName}
-                  name={`${colorName} - ${subColorName}`}
-                  color={subColorValue}
-                />
-              );
-            });
-          }
-          return <PaletteCard key={colorName} name={colorName} color={colorValue} />;
-        })}
-      </Grid>
+      <Wrapper>
+        <Grid>
+          {Object.entries(Theme.palette).map(palette => {
+            const colorName = palette[0];
+            const colorValue = palette[1];
+            if (typeof colorValue == 'function') {
+              return null;
+            }
+            if (typeof colorValue == 'object') {
+              return Object.entries(colorValue).map(subPalette => {
+                const subColorName = subPalette[0];
+                const subColorValue = subPalette[1];
```

Comment:
It's not really relevant but you can use array destructuration here instead of assigning  array index to table.